src/Symfony/Component/Security/Http/Event/InteractiveLoginFailureEvent.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+
+/**
+ * InteractiveLoginFailureEvent.
+ *
+ * @author Gabriel Ostrolucký <gabriel.ostrolucky@gmail.com>
+ */
+class InteractiveLoginFailureEvent extends Event
+{
+    private $request;
+    private $authenticationException;
+
+    /**
+     * Constructor.
+     *
+     * @param Request                 $request A Request instance
+     * @param AuthenticationException $ex
+     */
+    public function __construct(Request $request, AuthenticationException $ex)
```

Comment:
I was just following current convention. It's $ex in AuthenticationFailureEvent too